packages/RichTextEditing-Core.package/RichTextHTMLExportStrategy.class/instance/eventHandlingDictionary.st

```diff
@@ -0,0 +1,12 @@
+stop-condition-handling
+eventHandlingDictionary
+
+	eventHandlingDictionary ifNil: [
```

Comment:
Maybe @frcroth can answer why he introduced caching. Might be to avoid having to create a new dictionary with all the required method calls every time the dictionary is accessed. 
Besides, it is a instance dictionary in this case so it will be created once for every export function. You are right for the class side template dictionary caches though

Reply:
I didn't introduce caching here

Reply:
Ah, instance side, that's fine